Makefile

```diff
@@ -32,7 +32,11 @@ cargo-sort:
 	cargo install cargo-sort
 	cargo sort -c -w
 
-check: check-fmt check-clippy cargo-sort
+fmt-toml:
+	cargo install taplo-cli --locked
+	taplo fmt
```

Comment:
Sorry, I think this should be `taplo check` rather `taplo fmt` right?

Reply:
> Sorry, I think this should be `taplo check` rather `taplo fmt` right?

This job should perform `fmt` and we will have a check in CI.

Reply:
I mean, `fmt` does both check and formatting, will it return non 0 exit code when incorrectly formatted?

Reply:
> I mean, `fmt` does both check and formatting, will it return non 0 exit code when incorrectly formatted?

I'm guessing not. It will fmt the files directly if it used to be incorrectly formatted.

Reply:
> I'm guessing not. It will fmt the files directly if it used to be incorrectly formatted.

Than I think it's better to use `check` here. The goal of ci is to check the correctness, not do formatting.

Reply:
I can add a command to format and a command to check.